possible_topics.md

```diff
@@ -0,0 +1,51 @@
+# Possible topics for video content
+
+This document aims to collate possible ideas/outlines for video content
+related to onboarding (or otherwise encouraging) NumPy developers.
+
+## Workflow basics
+
+Start with the basic `git`/`github` workflow
+ - Fork, clone, set upstream
+
+Then on to setting up an environment
+ - There are a lot of options here (`conda`, `venv`, etc.) How to deal with the
+   various options?
+    * Personally, I like `venv` as it's Python standard library
+      and doesn't require any special knowledge of package managers
+ - Highlight the way `numpy` recommends doing it with the
+   `{test/doc}_requirements.txt` files.
+
+Installing numpy
+ - Again, various ways of doing it with their own advantages/disadvantages.
+   How to deal with the options? Is there a **right** way?
+     * Various `setupy.py` options (`install`, `develop`, `build_ext`, etc)
+     * `pip` and/or `conda`
+
+## Documentation build
+
+An obvious extension of `Workflow basics`, but has enough nuances to perhaps
+be its own separate thing (depending on how long/detailed the first is).
+For example:
+ - doc building checks the git hash, won't build if the current commit
+   differs from the installed version
+ - How to run doctests with `tools/refguide_check.py`?
+
+Walking through a documentation PR would be an obvious example, but maybe a
+separate video? Or could go into this one.
+
+## Running tests locally
+
+A video on `runtests.py` and the various features? Actually has a lot of
+(not-heavily-used) functionality like performing benchmarks with `asv` and
+type-checks with `mypy` (pretty new).
+
+## The full PR walkthrough
```

Comment:
@rossbar I like this idea. It could really help the broader FOSS community members to see this workflow. It could be more of a cooking show process at this point, if someone could screen record the 
1. PR submission
2. Respond to comments
Then, someone narrate the important steps. Its better if the PR is real, so the viewer can go see the finished product afterwards.

Reply:
Agreed! These are nice things to emphasize -adding to the doc...